skills/afk-fix/SKILL.md

```
@@ -0,0 +1,179 @@
+   BASE=$(gh pr view "$N" --repo "$REPO" --json baseRefName --jq .baseRefName) || BASE=""
+   [ -n "$BASE" ] || { echo "  ~ #$N: could not resolve base ref"; }   # empty → escalate below
+   git reset --hard && git clean -fd
+   gh pr checkout "$N" --repo "$REPO" --detach
+   git fetch origin "$BASE"    # /review --headless diffs `git diff origin/$BASE`; gh pr checkout fetches
+                               # the PR ref but not the base, so without this the base ref is stale/absent.
+   ```
+   If `$BASE` is empty, or the checkout or fetch fails, **escalate** (Step 4), reason `checkout/base-fetch failed`. `$BASE` feeds the re-review (step 4) and the publish re-derives the head branch itself, so a detached checkout is fine.
+
+4. **Convergence loop (max 3 rounds).** Goal: a local `/review --headless` with **no addressable findings**.
+   - **Round 1 — address posted feedback:** run the fix skill in defer mode —
+     ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)